src/test/java/RunWishListTest.java

```diff
@@ -0,0 +1,11 @@
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+
+@RunWith(Cucumber.class)
+@CucumberOptions(features = "resources/features", glue = "java/stepDefsForWishList", tags = "@CreateWishList")
+
+public class RunWishListTest {
```

Comment:
you should have one common Runner, which will run tests by @YourName tag.

Reply:
TestRun removed